src/gql/fetch/index.ts

```diff
@@ -0,0 +1,17 @@
+import { OperationDefinitionNode } from "graphql";
+import { SECRET_FIELDS } from "gql/queries";
+
+export const secretFieldsReq = {
```

Comment:
nit for a little bit more type safety
```suggestion
export const secretFieldsReq: RequestInit = {
```